parlai/crowdsourcing/tasks/model_chat/analysis/compile_results.py

```diff
@@ -15,6 +13,9 @@
 
 from parlai.crowdsourcing.utils.acceptability import AcceptabilityChecker
 from parlai.crowdsourcing.utils.analysis import AbstractTurnAnnotationResultsCompiler
+from parlai.crowdsourcing.tasks.model_chat.model_chat_blueprint import BLUEPRINT_TYPE
+
+_ = BLUEPRINT_TYPE
```

Comment:
Probably good to add a comment here saying that you're just importing BLUEPRINT_TYPE to force registration of the blueprint, but that you're not using this var itself